projects/plugins/jetpack/readme.txt

```diff
@@ -242,16 +242,19 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 11.2-a.3 - 2022-07-12
+### 11.2-a.5 - 2022-07-19
 #### Enhancements
-- Podcast Player: add new actions to make it possible for users to set up code that runs for podcast fetches.
-- VideoPress Block: add enhancements such as an upload progress bar, improved UX, and limited preview attempts (available in Beta only).
+- Blocks: enable Jetpack block collection for the native editor block inserter (on self hosted Jetpack sites).
+- Connection: make sure pre-existing settings are respected on plugin activation.
+- VideoPress: add UX improvements including fallback thumbnail replaced by loading element, and better error messaging (in Beta only). [#25088, #25064]
```

Comment:
```suggestion
- VideoPress: add UX improvements including fallback thumbnail replaced by loading element, and better error messaging (in Beta only).
```

Reply:
I've always wondered about that but just copied what I saw previously ([like here](https://github.com/Automattic/jetpack/pull/24674/files)). Better to remove do you think?

Reply:
We remove PR numbers from the readme file, right?

Reply:
Oh! This is readme. Sorry, haha. Will fix that.